client/v1/src/main/java/VelocloudApiClientV1.java

```diff
@@ -0,0 +1,131 @@
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+import org.opennms.velocloud.client.api.VelocloudApiClient;
+import org.opennms.velocloud.client.v1.api.AllApi;
+import org.opennms.velocloud.client.v1.api.ApiTokenApi;
+import org.opennms.velocloud.client.v1.api.ClientDeviceApi;
+import org.opennms.velocloud.client.v1.api.CompositeRoleApi;
+import org.opennms.velocloud.client.v1.api.ConfigurationApi;
+import org.opennms.velocloud.client.v1.api.CreateApi;
+import org.opennms.velocloud.client.v1.api.DeleteApi;
+import org.opennms.velocloud.client.v1.api.DisasterRecoveryApi;
+import org.opennms.velocloud.client.v1.api.EdgeApi;
+import org.opennms.velocloud.client.v1.api.EnterpriseApi;
+import org.opennms.velocloud.client.v1.api.EnterpriseProxyApi;
+import org.opennms.velocloud.client.v1.api.EventApi;
+import org.opennms.velocloud.client.v1.api.FirewallApi;
+import org.opennms.velocloud.client.v1.api.FunctionalRoleApi;
+import org.opennms.velocloud.client.v1.api.GatewayApi;
+import org.opennms.velocloud.client.v1.api.GetApi;
+import org.opennms.velocloud.client.v1.api.ImageApi;
+import org.opennms.velocloud.client.v1.api.LinkQualityEventApi;
+import org.opennms.velocloud.client.v1.api.LoginApi;
+import org.opennms.velocloud.client.v1.api.MetricsApi;
+import org.opennms.velocloud.client.v1.api.MigrationApi;
+import org.opennms.velocloud.client.v1.api.MonitoringApi;
+import org.opennms.velocloud.client.v1.api.MspApi;
+import org.opennms.velocloud.client.v1.api.NetworkApi;
+import org.opennms.velocloud.client.v1.api.OperatorApi;
+import org.opennms.velocloud.client.v1.api.OperatorUserApi;
+import org.opennms.velocloud.client.v1.api.PartnerApi;
+import org.opennms.velocloud.client.v1.api.ProxyApi;
+import org.opennms.velocloud.client.v1.api.RoleApi;
+import org.opennms.velocloud.client.v1.api.SystemApi;
+import org.opennms.velocloud.client.v1.api.SystemPropertyApi;
+import org.opennms.velocloud.client.v1.api.UpdateApi;
+import org.opennms.velocloud.client.v1.api.UserMaintenanceApi;
+import org.opennms.velocloud.client.v1.api.VcoDiagnosticsApi;
+import org.opennms.velocloud.client.v1.api.VcoInventoryApi;
+import org.opennms.velocloud.client.v1.api.VpnApi;
+import org.opennms.velocloud.client.v1.handler.ApiClient;
```

Comment:
Where are all of these defined? I can't find them anywhere.

Reply:
they are generated on every compilation with the swagger generator. 
You need to run `mvn clean install`
then for client/v1 they should be under `target/generated-sources/swagger/src/main/java/org/opennms/velocloud/client/v1`